### PR TITLE
Proper description of the homework 1And2 task in the README.md file

### PR DESCRIPTION
**Pull request by ellmaster**
# _**PROPER REVIEW NEEDED**_

In order to improve readability and to unify the location of task and homework there were changes made to the README.md file.

It does cover a detailed listing on what steps should be performed to get things done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,36 @@
...
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a358adf21c2442cd91a4827a50213ff1)](https://www.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Almantask/CSharp-From-Zero-To-Hero&amp;utm_campaign=Badge_Grade)  
-C# boot camp
+
+# Homework 1And2
+
+## Hints
+* The tasks require you to have already created an account on GitHub. Do that first!
+* If you do not understand something or need help:
+  * Ask questions in the channel bc-discussion in Discord.
+  * Check the wiki of this repository (work in progress!).
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 33
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536&bid=16212300)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,36 @@
...
+  * Check the bc-announcements channel for related stuff.
+
+## Task
+* Fork https://github.com/csinn/CSharp-From-Zero-To-Hero/tree/Chapter1/Homework/1And2
+  * Open the link.
+  * Click the fork button in the upper right.
+  * Click the green Clone or download button and clone your repository the way that suits you best.
+  * Open your repository in Visual Studio 2019.
+  * Select the right branch in the bottom right! That name should be Chapter1/Homework/1And2 instead of master in the end.
+  * Make sure that you open Bootcamp.sln. This is called a solution. Remember this!
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 3]
```diff
@@ -1,4 +1,36 @@
...
+* Source code from lesson 1 homework should end up on your forked branch.
+  * Open the Src folder.
+  * Open the BootCamp.Chapter project file. Remember that this is a project file!
+  * Open the Program.cs file.
+  * Put your code from homework 1 here.
+* Create a pull request to request your forked branch to be merged to the source branch.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 4]
```diff
@@ -1,4 +1,36 @@
...
+  * Click the fork button in the upper right.
+  * Click the green Clone or download button and clone your repository the way that suits you best.
+  * Open your repository in Visual Studio 2019.
+  * Select the right branch in the bottom right! That name should be Chapter1/Homework/1And2 instead of master in the end.
+  * Make sure that you open Bootcamp.sln. This is called a solution. Remember this!
+* Source code from lesson 1 homework should end up on your forked branch.
+  * Open the Src folder.
+  * Open the BootCamp.Chapter project file. Remember that this is a project file!
+  * Open the Program.cs file.
+  * Put your code from homework 1 here.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 5]
```diff
@@ -1,4 +1,36 @@
...
-[![Build Status](https://dev.azure.com/almantusk/From-Zero-To-Hero/_apis/build/status/Almantask.CSharp-From-Zero-To-Hero?branchName=master)](https://dev.azure.com/almantusk/From-Zero-To-Hero/_build/latest?definitionId=5&branchName=master)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a358adf21c2442cd91a4827a50213ff1)](https://www.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Almantask/CSharp-From-Zero-To-Hero&amp;utm_campaign=Badge_Grade)  
-C# boot camp
+
+# Homework 1And2
+
+## Hints
+* The tasks require you to have already created an account on GitHub. Do that first!
+* If you do not understand something or need help:
+  * Ask questions in the channel bc-discussion in Discord.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 6]
```diff
@@ -1,4 +1,36 @@
...
+  * Open the BootCamp.Chapter project file. Remember that this is a project file!
+  * Open the Program.cs file.
+  * Put your code from homework 1 here.
+* Create a pull request to request your forked branch to be merged to the source branch.
+  * Click the pencil symbol in the bottom right and type in a commit message.
+  * Press commit button.
+  * Click the single arrow button (next to the pencil) in the bottom right.
+  * Push your commit.
+  * Click the line plus arrow button (next to the single arrow) in the bottom right.
+  * Click create new element.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 7]
```diff
@@ -1,4 +1,36 @@
...
+  * Check the wiki of this repository (work in progress!).
+  * Check the bc-materials channel for reference material.
+  * Check the bc-announcements channel for related stuff.
+
+## Task
+* Fork https://github.com/csinn/CSharp-From-Zero-To-Hero/tree/Chapter1/Homework/1And2
+  * Open the link.
+  * Click the fork button in the upper right.
+  * Click the green Clone or download button and clone your repository the way that suits you best.
+  * Open your repository in Visual Studio 2019.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 8]
```diff
@@ -1,4 +1,36 @@
...
+  * Select the right branch in the bottom right! That name should be Chapter1/Homework/1And2 instead of master in the end.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 9]
```diff
@@ -1,4 +1,36 @@
...
-C# boot camp
+
+# Homework 1And2
+
+## Hints
+* The tasks require you to have already created an account on GitHub. Do that first!
+* If you do not understand something or need help:
+  * Ask questions in the channel bc-discussion in Discord.
+  * Check the wiki of this repository (work in progress!).
+  * Check the bc-materials channel for reference material.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 10]
```diff
@@ -1,4 +1,36 @@
...
+  * Make sure that you open Bootcamp.sln. This is called a solution. Remember this!
+* Source code from lesson 1 homework should end up on your forked branch.
+  * Open the Src folder.
+  * Open the BootCamp.Chapter project file. Remember that this is a project file!
+  * Open the Program.cs file.
+  * Put your code from homework 1 here.
+* Create a pull request to request your forked branch to be merged to the source branch.
+  * Click the pencil symbol in the bottom right and type in a commit message.
+  * Press commit button.
+  * Click the single arrow button (next to the pencil) in the bottom right.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 11]
```diff
@@ -1,4 +1,36 @@
...
+* Fork https://github.com/csinn/CSharp-From-Zero-To-Hero/tree/Chapter1/Homework/1And2
+  * Open the link.
+  * Click the fork button in the upper right.
+  * Click the green Clone or download button and clone your repository the way that suits you best.
+  * Open your repository in Visual Studio 2019.
+  * Select the right branch in the bottom right! That name should be Chapter1/Homework/1And2 instead of master in the end.
+  * Make sure that you open Bootcamp.sln. This is called a solution. Remember this!
+* Source code from lesson 1 homework should end up on your forked branch.
+  * Open the Src folder.
+  * Open the BootCamp.Chapter project file. Remember that this is a project file!
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 12]
```diff
@@ -1,4 +1,36 @@
...
+* Create a pull request to request your forked branch to be merged to the source branch.
+  * Click the pencil symbol in the bottom right and type in a commit message.
+  * Press commit button.
+  * Click the single arrow button (next to the pencil) in the bottom right.
+  * Push your commit.
+  * Click the line plus arrow button (next to the single arrow) in the bottom right.
+  * Click create new element.
+  * Click on master branch and change it to Chapter1/Homework/1And2.
+  * Give your pull request a meaningful title and description.
+  * Click create.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 13]
```diff
@@ -1,4 +1,36 @@
 # CSharp-From-Zero-To-Hero
-[![Build Status](https://dev.azure.com/almantusk/From-Zero-To-Hero/_apis/build/status/Almantask.CSharp-From-Zero-To-Hero?branchName=master)](https://dev.azure.com/almantusk/From-Zero-To-Hero/_build/latest?definitionId=5&branchName=master)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a358adf21c2442cd91a4827a50213ff1)](https://www.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Almantask/CSharp-From-Zero-To-Hero&amp;utm_campaign=Badge_Grade)  
-C# boot camp
+
+# Homework 1And2
+
+## Hints
+* The tasks require you to have already created an account on GitHub. Do that first!
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 14]
```diff
@@ -1,4 +1,36 @@
...
+  * Open the Program.cs file.
+  * Put your code from homework 1 here.
+* Create a pull request to request your forked branch to be merged to the source branch.
+  * Click the pencil symbol in the bottom right and type in a commit message.
+  * Press commit button.
+  * Click the single arrow button (next to the pencil) in the bottom right.
+  * Push your commit.
+  * Click the line plus arrow button (next to the single arrow) in the bottom right.
+  * Click create new element.
+  * Click on master branch and change it to Chapter1/Homework/1And2.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

File: README.md [hunk 15]
```diff
@@ -1,4 +1,36 @@
...
+* Source code from lesson 1 homework should end up on your forked branch.
+  * Open the Src folder.
+  * Open the BootCamp.Chapter project file. Remember that this is a project file!
+  * Open the Program.cs file.
+  * Put your code from homework 1 here.
+* Create a pull request to request your forked branch to be merged to the source branch.
+  * Click the pencil symbol in the bottom right and type in a commit message.
+  * Press commit button.
+  * Click the single arrow button (next to the pencil) in the bottom right.
+  * Push your commit.
```
Comment (Almantask) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)

**Comment by RoelofWobben** [comment]
Can I maybe help with reviewing or is this help wanted meant for Kai

**Comment by ellmaster** [comment]
> Can I maybe help with reviewing or is this help wanted meant for Kai

You can help as well! :)
If you test the things or verify them as working or point out things that are wrong, this would be perfect. :)

**Comment by RoelofWobben** [comment]
looks welll to me

**Comment by ellmaster** [comment]
From feedback it looks fine. You can merge if you want to.

**Pull request merged**